tensorflow/g3doc/api_docs/python/client.md

```diff
@@ -84,7 +84,7 @@ the session constructor.
 
 
 *  <b>`target`</b>: (Optional.) The execution engine to connect to.
-    Defaults to using an in-process engine. See [Distributed Tensorflow]
```

Comment:
This is autogenerated from the python documentation, so you'll want to edit the version there instead and revert this file